etl/pad/raw/dungeon.py

```diff
@@ -3,22 +3,80 @@
 """
 
 import csv
-import re
 from io import StringIO
-from typing import Any, List, Optional
+from typing import Any, Dict, List, Optional, Set, Union
 
 from pad.common import pad_util
 from pad.common.dungeon_types import RawDungeonType, RawRepeatDay
+from pad.common.pad_util import ghtime
 from pad.common.shared_types import DungeonId, SubDungeonId
 
 # The typical JSON file name for this data.
 FILE_NAME = 'download_dungeon_data.json'
 
 
+def maybe_int(num: str) -> Union[str, int]:
```

Comment:
random mini utils like this should go into a common mini utils file elsewhere called like `number.py` or something

probably not tsutils cos thats just tsubaki but you can have a `/common` folder somewhere in the pipeline

don't necessarily need to fix this now for this PR but if you want to you could, but in general yeah don't put sth like this in this file